packages/utils/crates/utils-types/src/primitives.rs

```diff
@@ -56,11 +59,17 @@ impl Address {
 
     /// Creates a random Ethereum address, mostly used for testing
     pub fn random() -> Self {
+        // Uses getrandom, because it cannot bring in dependency on core-crypto
```

Comment:
This will be possible once the core and core-ethereum meta peckages are flattened.

Reply:
There's a different idea that I had a long time ago on unfiying these randomness under a single core-crypto trait